compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangParserListener.java

```diff
@@ -2174,25 +2174,33 @@ public void exitSimpleLiteral(BallerinaParser.SimpleLiteralContext ctx) {
         Object value;
         BallerinaParser.IntegerLiteralContext integerLiteralContext = ctx.integerLiteral();
         if (integerLiteralContext != null && (value = getIntegerLiteral(ctx, ctx.integerLiteral())) != null) {
-            this.pkgBuilder.addLiteralValue(pos, ws, TypeTags.INT, value);
+            this.pkgBuilder.addLiteralValue(pos, ws, TypeTags.INT, value,
+                                            getOriginalIntegerValue(integerLiteralContext));
```

Comment:
is ctx.getText() giving the orginal string value. ?
```suggestion
                                            ctx.getText());
```

Reply:
Yes ctx.getText() gives the original string value. Fixed it with 305aa04. Thanks :)

Reply:
Sorry seems like the change was not committed properly :( Updated the PR with cee2b10